### ISSUE TITLE
SortedSwitchCases Bug

### ISSUE THREAD

**Issue by PompeiaPaetenari** [nicklockwood/SwiftFormat#821]
With my massive PR to introduce this rule to our code, one of the reviewers found this incorrectly sorted case:

```swift
    case (_, .undetermined),
         (_, .unavailable),
         (_, .enrolled),
         (.delta, _),
         (.other, _):
```
The first parameter is alphabetical, and then the second is reverse alphabetical.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari thanks for the heads up - I suspect that tuple cases like this aren't handled correctly (or at all?)

**Comment by PompeiaPaetenari** [comment]
A lot of our tuples are sorted correctly, perhaps happy accidents because the second or third parameters coincide with the alphabetical sort of the first. xD

**Comment by nicklockwood** [comment]
@PompeiaPaetenari fixed in 0.47.6
